debezium-connector-mysql/src/test/java/io/debezium/connector/mysql/MySqlConnectorIT.java

```diff
@@ -309,6 +326,15 @@ public void shouldValidateLockingModeNoneWithValidSnapshotModeConfiguration() {
 
     @Test
     public void shouldConsumeAllEventsFromDatabaseUsingSnapshot() throws SQLException, InterruptedException {
+        shouldConsumeAllEventsFromDatabaseUsingSnapshotByField(MySqlConnectorConfig.DATABASE_INCLUDE_LIST, 18765);
+    }
+
+    @Test
+    public void shouldConsumeAllEventsFromDatabaseUsingSnapshotOld() throws SQLException, InterruptedException {
+        shouldConsumeAllEventsFromDatabaseUsingSnapshotByField(MySqlConnectorConfig.DATABASE_WHITELIST, 18775);
+    }
+
```

Comment:
instead of duplicating the big test case I decided to parametrize the original test and call it with the 2 different parameters